site/_posts/2021-12-10-log4j-fusionauth.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: blog-post
+title: "log4j CVE: How it affects FusionAuth (TLDR: It doesn't)"
+description: FusionAuth does not use log4j and is not affected by CVE-2021-44228
+author: Dan Moore
+image: blogs/fusionauth-log4j-cve/log4j-cve-how-it-affects-fusionauth-tldr-it-doesnt-header-image.png
+category: blog
+excerpt_separator: "<!--more-->"
+---
+
+The recent announcement of [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), which allows for "arbitrary code loaded from LDAP servers when message lookup substitution is enabled" through a vulnerability in Log4J has many people double checking the dependencies of their Java applications.
```

Comment:
Just a nit..  most places you are referencing `log4j`, and here you have `Log4J`. Not sure if that is on purpose.